modules/oe_theme_content_event/oe_theme_content_event.module

```diff
@@ -21,6 +21,8 @@ function oe_theme_content_event_theme() {
         'label' => '',
         'url' => '',
         'description' => '',
+        'approaching_time_description' => '',
+        'approaching_time' => FALSE,
```

Comment:
This name is a bit confusing. How about we call it "registration_day" and "registration_day_description"?